hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/benefits/BenefitInformationController.java

```diff
@@ -69,6 +69,9 @@ public String viewBenefitInfo(ModelMap model, PortletRequest request) {
       model.addAttribute("isMadisonUser", isMadisonUser);
       final PortletPreferences preferences = request.getPreferences();
       model.addAttribute("learnMoreEBenefitGuide", preferences.getValue("ebenefitguidetext", null));
+      model.addAttribute("madisonNewLearnMore"), "https://hr.wisc.edu/benefits/new-employee-benefits-enrollment/";
```

Comment:
I wouldn't do this. If these are going to be hard-coded values used exactly once in a JSP, I'd just go ahead and hard-code the value in the JSP. Fewer layers to get right, same effect, same effort and shape of change (.war release, migration) to change in future. Maybe even better in that we could probably decide to hack a JSP in an overlay if it came to it.